source/ExpandablePicker.js

```diff
@@ -553,6 +553,25 @@
 			}
 			this.$.client.setHighlander(!this.multipleSelection);
 			this.selectedChanged();
+		},
+
+		/**
+		* If drawer is currently open, and event was sent via keypress (i.e., it has a direction),
+		* process header focus.
+		*
+		* @fires moon.Scroller#onRequestScrollIntoView
+		* @private
+		*/
+		headerFocus: function (inSender, inEvent) {
```

Comment:
I don't like effectively duplicating the exact same logic from `moon.ExpandableListItem` here. As a potential alternative, we could do something like this: https://github.com/enyojs/moonstone/compare/ENYO-2110-kunmyonchoi-2.5-new...ENYO-2110-aarontam